ClientApp/src/app/news-feed/news-feed.component.scss

```diff
@@ -0,0 +1,16 @@
+.news-thumbnail {
+  width: 220px;
+  height: 123px;
+}
+
+.news-block {
+  color: initial;
+}
+h5 { 
+color: red;
+}
+
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6318563)